### PR TITLE
cs: keep class spacing consistent

### PR DESCRIPTION
**Pull request by TomasVotruba**
| Q             | A
| ------------- | ---
|Description, reason for the PR| Empty lines in classes are random at the moment, 0-1 at the beggining, 0-1 at the end, 1-2 between class elements. This PR adds `ClassAttributesSeparationFixer` that makes it consistent and fixes the code to comly with it.
|New feature| No
|BC breaks| No
|Standards and tests pass| Yes
|Have you read and signed our [License Agreement for contributions](https://www.shopsys.com/license-agreement)?| Yes

It would make code more consistent and proffesional. People would not have to decide/think about spacing in classes and would be more likely to contribute us.

Found with fixer:

```yaml
# easy-coding-standard.yml
services:
    PhpCsFixer\Fixer\ClassNotation\ClassAttributesSeparationFixer:
        elements:
            - 'property'
            - 'method'
```

What do you think?

### Todo

- [x] waits on #371
- [x] retarget to `master` after it's merged

### CODE REVIEW

**Comment by TomasVotruba** [comment]
What needs to be done here? It's seem 4 people from the team agree.

**Comment by vitek-rostislav** [comment]
Hi @TomasVotruba. So far, there is nothing that needs to be done on your side (maybe besides adding entry into `CHANGELOG.md` but I if you do not feel like doing that, don't bother, I'll do that for you as the changelog will be generated automatically very soon hopefully :innocent:) 

4 people agree that this is a relevant PR, thanks to that we decided to solve it in the current sprint. But I need to review the code yet so please be a little bit more patient, I`ll work on it today :wink:

**Comment by TomasVotruba** [comment]
I see, thanks for update :)

**Comment by vitek-rostislav** [comment]
Hi @TomasVotruba, I have reviewed and tested your code and it is ok with me. The only two problems are that is not up-to-date with current `master` and also one file (`packages/framework/src/Command/CheckMicroservicesCommand`) is now failing to pass the new check. Please fix it :innocent:

**Comment by TomasVotruba** [comment]
Hey, what checks should I use?

I've run these 2 and there was no error:

```bash
php phing standards-packages -debug
php phing ecs -debug
```

I might fix it in the past and I don't remember though.

**Comment by TomasVotruba** [comment]
In the end used classic direct command, because I don't use Docker and I don't have Redis installed - both are somehow requirements for `php phing standards-packages -debug` (not sure why).

```
vendor/bin/ecs check project-base/src project-base/tests packages
```

**Comment by vitek-rostislav** [comment]
Hi, thanks :+1: 

The task is simply
```php phing standards```

It is ok now, I have added the commit that adds information in the `CHANGELOG.md`, now I need yet another one team member to review this PR, afterwards we could merge it.

File: CHANGELOG.md [hunk 1]
```diff
@@ -100,7 +100,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
        
 #### Changed
 - [#389 - Standardized YAML files indentation to 4 spaces](https://github.com/shopsys/shopsys/pull/389)
-              
+
+### [shopsys/coding-standards]
```
Comment (boris-brtan) [inline CHANGELOG.md]: this should be moved to unreleased section above alpha5 version

File: packages/coding-standards/easy-coding-standard.yml [hunk 2]
```diff
@@ -6,6 +6,14 @@ services:
     # Slevomat Checkers
     SlevomatCodingStandard\Sniffs\Classes\UnusedPrivateElementsSniff: ~
 
+    # keep 1 empty line between constants, properties and methods
```
Comment (boris-brtan) [inline packages/coding-standards/easy-coding-standard.yml]: consider to reposition this into # Custom Shopsys standards section

**Comment by vitek-rostislav** [comment]
Hi @boris-brtan, thanks for your notes, I'll do the changes, you do not need to bother @TomasVotruba :wink:

**Comment by boris-brtan** [comment]
nice coop @TomasVotruba and @vitek-rostislav. It can go to tests.

**Comment by vitek-rostislav** [comment]
Hi, I have already tested it and it also passes Jenkins CI build -> merge :slightly_smiling_face:

**Pull request merged**